### PR TITLE
Add CSRF support

### PR DESCRIPTION
**Pull request by johnkord**

### CODE REVIEW

**Review by admangum — DISMISSED** [review]
Looks good to me!

**Comment by johnkord** [comment]
@davideschiera Can you merge this?

File: bower.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "d3": "4.11.0",
     "mousetrap": "1.6.1",
     "normalize-css": "7.0.0",
-    "oboe": "2.1.4"
+    "oboe": "2.1.4",
+    "js-cookie": "2.2.1"
```
Comment (admangum) [inline bower.json:17]: @davideschiera we're adding this dependency here, even though it **is not** referenced in this repo's codebase but rather is referenced in sysdig-inspect-web [here](https://github.com/draios/sysdig-inspect-web/pull/2/files#diff-6d57078a817396eafc021282e6c8b2db).
Is this approach as designed?

File: ember-cli-build.js [hunk 2]
```diff
@@ -29,6 +29,11 @@ module.exports = function(defaults) {
     app.import(app.bowerDirectory + '/mousetrap/mousetrap.js');
     app.import(app.bowerDirectory + '/normalize-css/normalize.css');
     app.import(app.bowerDirectory + '/oboe/dist/oboe-browser.js');
+    app.import(app.bowerDirectory + '/js-cookie/src/js.cookie.js', {
+        using: [
+            { transformation: 'amd', as: 'js-cookie' }
+        ],
+    });
```
Comment (admangum) [inline ember-cli-build.js:36]: @johnkord if this is the right approach, I think we should definitely add a comment here explaining why we're importing js-cookie. Without knowledge of sysdig-inspect-web, there's no way to know why this is needed.

**Review by davideschiera — APPROVED** [review]
Part of the new code could be refactored and move outside this project, but this can happen as a second step.

File: bower.json (hunk 1 above)
Comment (davideschiera) [reply bower.json:17]: Given the current state, this is now _more_ correct. All good!

File: ember-cli-build.js (hunk 2 above)
Comment (davideschiera) [reply ember-cli-build.js:36]: Ideally, we should find a way to remove the dependency entirely. Otherwise it's a good point.

**Pull request merged**